src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/FormattingTestBase.cs

```diff
@@ -183,16 +185,43 @@ private static (RazorCodeDocument, DocumentSnapshot) CreateCodeDocumentAndSnapsh
         {
             fileKind ??= FileKinds.Component;
             tagHelpers ??= Array.Empty<TagHelperDescriptor>();
+            if (fileKind == FileKinds.Component)
+            {
+                tagHelpers = tagHelpers.Concat(GetDefaultRuntimeComponents()).ToArray();
```

Comment:
I'd break the `GetDefaultRuntimeComponents` into its own local so it's easy to debug for future onlookers.